Fixed Function esc_html in main.page.php

DAnn2012 opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
<!-- Would you like this feature to be tested by Beta testers?
Please add testing instructions to projects/plugins/jetpack/to-test.md in a new commit as part of your PR. -->
<!-- a12s: If you have an expected version that you're aiming for the PR to add, please use the Milestone field to communicate it. If you leave it blank, that indicates there isn't a preference. -->

Fixes #

## Proposed changes:
<!--- Explain what functional changes your PR includes -->
* Fixed Function esc_html in main.page.php

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion
<!-- If you're an Automattician, include a shortlink to the p2 discussion with Jetpack Product here. -->
<!-- Make sure any changes to existing products have been discussed and agreed upon -->

## Does this pull request change what data or activity we track or use?
<!--- If so, please add the "[Status] Needs Privacy Updates" label and explain what changes there are. -->
<!--- Check existing Jetpack support documents for a preview of the information we need. -->

## Testing instructions:
<!-- If you were reviewing this PR, how would you like the instructions to be presented? -->
<!-- Please include detailed testing steps, explaining how to test your change. -->
<!-- Bear in mind that context you working on is not obvious for everyone.  -->
<!-- Adding "simple" configuration steps will help reviewers to get to your PR as quickly as possible. -->
<!-- "Before / After" screenshots can also be very helpful when the change is visual. -->

* Go to '..'
*

DAnn2012 committed:
Update main.page.php

manzoorwanijk reviewed (dismissed):
Thank you for your contribution.

DAnn2012 committed:
Add changelog file

DAnn2012 committed:
Update fix-esc-html-wrapper-export-heading

manzoorwanijk commented:
The PR needs a trunk merge before we can proceed further.

DAnn2012 committed:
Merge branch 'Automattic:trunk' into patch-1

DAnn2012 committed:
Merge branch 'Automattic:trunk' into patch-1

DAnn2012 committed:
Update main.page.php

DAnn2012 committed:
Merge branch 'Automattic:trunk' into patch-1

kraftbj committed:
Merge remote-tracking branch 'origin/trunk' into DAnn2012-patch-1

kraftbj commented:
@tbradsha I merged trunk since it's on us that we haven't landed this in a timely way.

kraftbj committed:
Add translators comment for export heading placeholders

Fixes PHPCS WordPress.WP.I18n.MissingTranslatorsComment violation
on the sprintf __() call with %1$s/%2$s/%3$s placeholders.

tbradsha reviewed (dismissed):
Thanks! Looks good.

kraftbj committed:
Fix typo in text domain for Go Back button

The text domain was 'zerob-bs-crm' instead of 'zero-bs-crm',
preventing the string from being translated.
